### ISSUE TITLE
What is my SCIM Endpoint URL?

### ISSUE THREAD

**Issue by iamexpertguy** [MicrosoftDocs/entra-docs#532]
[Enter feedback here]
Im new here. I followed all the instructions and created a sample app. But, the first q of the SCIM vcalidator is: SCIM Endpoint URL. And then, Token. How and where can I get these values?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 37753d44-c812-b775-134c-e1081f8a2cd9
* Version Independent ID: 05b3ccf1-e811-0498-c0c4-e91cd2034606
* Content: [Tutorial - Test your SCIM endpoint for compatibility with the Microsoft Entra provisioning service. - Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/identity/app-provisioning/scim-validator-tutorial)
* Content Source: [docs/identity/app-provisioning/scim-validator-tutorial.md](https://github.com/MicrosoftDocs/entra-docs/blob/main/docs/identity/app-provisioning/scim-validator-tutorial.md)
* Service: **entra-id**
* Sub-service: **app-provisioning**
* GitHub Login: @kenwith
* Microsoft Alias: **kenwith**

**Comment by Naveenommi-MSFT** [comment]
@iamexpertguy 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by ManoharLakkoju-MSFT** [comment]
Hi @iamexpertguy 
To find your SCIM Endpoint URL, you need to check the documentation or configuration of your app. The SCIM Endpoint URL is the URL of the endpoint that your app exposes to receive SCIM requests from Azure AD. 

If you have followed the tutorial "Build a SCIM endpoint for user provisioning to apps from Azure Active Directory" (https://docs.microsoft.com/en-us/azure/active-directory/app-provisioning/use-scim-to-build-users-and-groups-endpoints), then your SCIM Endpoint URL should be the URL of your app's endpoint that you deployed to Azure App Service. 

You can find the URL of your app's endpoint in the Azure portal by going to your app's **Overview** page and looking for the **URL** field. The SCIM Endpoint URL should be the URL of your app's endpoint with "/scim" appended to it.

For example, if your app's endpoint URL is "https://myapp.azurewebsites.net", then your SCIM Endpoint URL should be "https://myapp.azurewebsites.net/scim".

Once you have your SCIM Endpoint URL, you can use it to test your SCIM endpoint with the Azure AD SCIM Validator.

**Comment by iamexpertguy** [comment]
Hi,

Thanks for the answer. I added followed your description adding the /scim at the end of the app but still I failed:
***@***.***





Then, I tried to change the key value in  ‘appsettings.Development.json’ and rebuild the solution. It failed with a lot of errors:

***@***.***

I used Tools to select ALL asp.net and web development components to update Visual Studio 2022 hoping the missing packages are there.

***@***.***




Then I tried to rebuild again. It showed the same errors.


Then I just published it

Publishing failed. I cannot remember if it failed the first time.

***@***.***




So now I’m going to repeat all the steps again.

I cloned the URL AzureAD/SCIMReferenceCode: Reference code to build a SCIM endpoint to automate provisioning (github.com)<https://github.com/AzureAD/SCIMReferenceCode> to a new folder

I changed the key and app name published. Last time I just used the default values.

***@***.***


***@***.***

Token__TokenIssuer: https://sts.windows.net/40e82f81-9046-4602-93ee-dc72555b2347/

Are you aware of that the string ‘Token__TokenIssuer’ contains two (2) anderscores? Is it correct?

***@***.***

I left the deployment slot setting unchecked.

Then I filled the scimvalidator with https://microsoftscimwebhostsampleiamexpert2.azurewebsites.net/scim and tested. The same failure.

***@***.***



The app is running:
***@***.***







My key value is “12zakmes34”  . You are welcome to try.

I hope you can understand what’s not working…


Greetings,  Guy

**Comment by ManoharLakkoju-MSFT** [comment]
@iamexpertguy 
I'd recommend working closer with our support team via an [Azure support request] (https://docs.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request). Or you can leverage our Q&A forum by posting your issue there so our community, and MVPs can further assist you in troubleshooting this issue or finding potential workarounds.

[Teams Q&A forum] (https://docs.microsoft.com/en-us/answers/topics/46488/office-teams-windows-itpro.html) for technical questions about the configuration and administration of Microsoft Teams on Windows.
[Microsoft Teams Community forum] (https://answers.microsoft.com/en-us/msteams/forum?sort=LastReplyDate&dir=Desc&tab=All&status=all&mod=&modAge=&advFil=&postedAfter=&postedBefore=&threadType=All&isFilterExpanded=false&page=1)
Thank you for your time and patience throughout this issue.

**Comment by iamexpertguy** [comment]
I don’t agree. The whole procedure do not work. I invested my time in it so I expect Microsoft to deliver something working and stand behind it.

So I still expect a concrete answer to my  detailed and well documented issue.

Greeting, Guy
